config/admin-functions/show-vx-suite-admin-menu.sh

```diff
@@ -12,6 +12,12 @@ if [[ $(tty) = /dev/tty1 ]] && [[ -f "/home/REKEY_VIA_TPM" ]]; then
   sudo "${VX_FUNCTIONS_ROOT}/rekey-via-tpm.sh"
 fi
 
+# We do this after the rekey operation to save time. It's much faster to encrypt /var
+# before we expand it to the max available storage.
+if [[ $(tty) = /dev/tty1 ]]; then
+  sudo "${VX_FUNCTIONS_ROOT}/expand-var-filesystem.sh"
```

Comment:
I'm confused, we're expanding after rekeying, but in the expansion we're using `insecure` as the passphrase?

Reply:
If it's a non-secure image, we don't rekey, so it uses `insecure` as the passphrase. If it's a secure image, the `crypttab` config will cause it to use the TPM instead, and the passphrase will be ignored.

Reply:
ahhh, right, thank you.